build/generateLicenses.js

```diff
@@ -24,29 +26,53 @@ const customFormat = {
   copyright: "",
   licenseFile: "none",
   licenseText: "none",
-  licenseModified: "no"
+  licenseModified: "no",
 };
 
 const customFormatFile = tmp.fileSync();
 fs.writeFileSync(customFormatFile.name, JSON.stringify(customFormat));
 
+const disallowedLicenses = [
+  "GPL",
+  "GPL-2.0",
+  "GPL-3.0",
+  "AGPL",
+  "LGPL",
+  "LGPL-3.0",
+  "NGPL",
+  "MIT",
+];
```

Comment:
あ、LGPLとかMITは大丈夫なのでこうかな？
```suggestion
const disallowedLicenses = [
  "GPL",
  "GPL-2.0",
  "GPL-3.0",
  "AGPL",
  "NGPL",
];
```